package.json

```diff
@@ -23,6 +23,7 @@
     "babel-preset-env": "^1.6.1",
     "babel-preset-react": "^6.24.1",
     "babel-preset-stage-1": "^6.24.1",
+    "benchmark": "^2.1.4",
```

Comment:
We import this but did not list it so the linter was throwing errors, adding it to devDep